coverage.outc

```diff
@@ -0,0 +1 @@
+mode: set
```

Comment:
wat is this

Reply:
oops its a go tool coverage report I forgot to delete